### DISCUSSION TITLE
Bot can't see adding new users into group

### DISCUSSION THREAD

**Discussion by savelmtr** [eternnoir/pyTelegramBotAPI#2140] [Q&A] [answered]
python:3.10
pyTelegramBotAPI 4.14.1
OS = Debian buster

When I try to catch adding new member in a group, I get nothing. What I've tried:
1) `message_handler(content_types=['new_chat_member'])`
2) `chat_member_handler(lambda x: True)`

My bot has administrative rights in the group.

What have I done wrong?

**Comment by Badiboy** [comment, marked as answer]
content_types=['new_chat_members']

**Reply by savelmtr** [reply]
Thank you, this is just what I am searching for.
